### PR TITLE
Apollo and yoga examples on subscriptions

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
Simple examples on setting up GraphQL subscriptions in Apollo and Yoga

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
| name | dbHits | old dbHits | time (ms) | old time (ms) | maxRows |
| ---- | ------ | ---------- | --------- | ------------- | ------- |
|  aggregations.TopLevelAggregate | 3403 | 3403 | 32 | 54 | 1134 |
|  aggregations.NestedAggregation | 3403 | 3403 | 230 | 387 | 1134 |
|  aggregations.AggregationWithWhere | 13210 | 13210 | 67 | 83 | 2174 |
|  aggregations.NestedCountFromMovieToActors | 8694 | 8694 | 66 | 97 | 2174 |
|  aggregations.NestedCountFromActorsToMovie | 8900 | 8900 | 62 | 114 | 2174 |
|  aggregations.DeeplyNestedCount | 10054408 | 10054408 | 4554 | 5194 | 2008534 |
|  batch-create.BatchCreate | 3600 | 3600 | 93 | 212 | 600 |
|  connect.createAndConnect | 12419 | 12419 | 151 | 342 | 3003 |
|  connections.Connection | 13042 | 13042 | 246 | 214 | 2174 |
|  connections.NestedConnection | 33883 | 33883 | 128 | 132 | 4516 |
|  create.SimpleMutation | 6 | 6 | 41 | 43 | 1 |
|  cypher-directive.TopLevelCypherDirective | 99525 | 99525 | 700 | 875 | 10168 |
|  cypher-directive.TopLevelCypherDirectiveWithColumnName | 127994 | 127994 | 259 | 291 | 10168 |
|  delete.SimpleDelete | 18361 | 18361 | 280 | 276 | 1040 |
|  delete.NestedDeleteInUpdate | 16779 | 16779 | 183 | 218 | 2040 |
|  query.SimpleQuery | 14120 | 14120 | 198 | 216 | 2174 |
|  query.QueryWhere | 13018 | 13014 | 36 | 44 | 2171 |
|  query.SimpleQueryWithNestedWhere | 13189 | 13185 | 58 | 54 | 2171 |
|  query.OrFilterOnRelationships | 51294 | 48282 | 138 | 196 | 1849 |
|  query.OrFilterOnRelationshipsAndNested | 38706 | 36048 | 193 | 218 | 1849 |
|  query.QueryWithNestedIn | 15479 | 14744 | 50 | 65 | 1366 |
|  query.NestedConnectionWhere | 13182 | 13182 | 58 | 59 | 2174 |
|  query.DeeplyNestedConnectionWhere | 13266 | 13291 | 75 | 80 | 2174 |
|  query.DeeplyNestedWithRelationshipFilters | 6913 | 6913 | 119 | 129 | 1134 |
|  query.DeeplyNestedWithRelationshipSingleFilters | 15438 | 15438 | 102 | 129 | 2174 |
|  query.Fulltext | 96 | 96 | 38 | 33 | 16 |
|  query.FulltextWithNestedQuery | 571 | 571 | 41 | 46 | 84 |
|  sorting.SortMultipleTypes | 2493 | 2493 | 61 | 71 | 1040 |
|  sorting.SortMultipleTypesWithCypherWithCypher | 1464 | 1404 | 166 | 232 | 1040 |
|  sorting.SortOnNestedFields | 13042 | 13042 | 80 | 76 | 2174 |
|  sorting.SortDeeplyNestedFields | 38965 | 38965 | 143 | 175 | 4516 |
|  unions.SimpleUnionQuery | 321 | 321 | 52 | 59 | 35 |
|  unions.SimpleUnionQueryWithMissingFields | 293 | 293 | 51 | 57 | 35 |
|  unions.NestedUnion | 398553 | 398553 | 270 | 341 | 33033 |
|  unions.NestedUnionWithMissingFields | 372527 | 372527 | 263 | 348 | 33033 |

Old Schema Generation: 26.613s
Schema Generation: 27.238s

File: examples/subscriptions/yoga_sse/server.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    },
+});
+
+// Generates graphql schema and resolvers, connect to neo4j
+neoSchema.getSchema().then((schema) => {
+    const yoga = createYoga({
+        schema,
+    });
+
+    const server = createServer(yoga);
```
Comment (darrellwarde) [inline examples/subscriptions/yoga_sse/server.js:52]: Wow, this is remarkably simple compared to Apollo! 👀

**Review by darrellwarde — APPROVED** [review]
LGTM! At some point in the future, it would be good to see E2E tests written for the Yoga case, and also to see a tutorial or two for these specific Subscriptions deployments in our documentation.

File: examples/subscriptions/yoga_sse/server.js (hunk 1 above)
Comment (angrykoala) [reply examples/subscriptions/yoga_sse/server.js:52]: Yep, clearly yoga had subscriptions in mind from the beginning

**Pull request merged**